source/extensions/filters/network/local_ratelimit/local_ratelimit.h

```diff
@@ -78,13 +78,16 @@ class Config : Logger::Loggable<Logger::Id::filter> {
   bool canCreateConnection();
   bool enabled() { return enabled_.enabled(); }
   LocalRateLimitStats& stats() { return stats_; }
+  const absl::optional<std::chrono::milliseconds> delay() const { return delay_; }
 
 private:
   static LocalRateLimitStats generateStats(const std::string& prefix, Stats::Scope& scope);
 
   LocalRateLimiterImplSharedPtr rate_limiter_;
   Runtime::FeatureFlag enabled_;
   LocalRateLimitStats stats_;
+  absl::optional<std::chrono::milliseconds> delay_;
+  std::vector<RateLimit::LocalDescriptor> descriptors_;
```

Comment:
Is this `descriptors_` necessary?

Reply:
thanks for pointing it out. I made the changes based on a very old version. Will update